src/nisarqa/raster_classes.py

```diff
@@ -373,23 +423,23 @@ class RadarGrid(CoordinateGrid):
         1D vector of zero Doppler azimuth times (in seconds) measured relative
         to a UTC epoch. These correspond to the center of each pixel
         of the raster grid in the Y direction.
-    zero_doppler_time_spacing : float
+    zero_doppler_time_posting : float
```

Comment:
Let's just call this `spacing` instead of `posting`. It's common to just use "spacing" when referring to the radar grid, since the posting is always positive. I've only ever heard folks use "posting" to refer to geocoded grids.